znoyder/config.yml

```diff
@@ -178,7 +178,7 @@ add:
             # yamllint disable-line rule:line-length
             - "sed -i -r 's/swift_hash_path_suffix = %SWIFT_HASH_PATH_SUFFIX%/swift_hash_path_suffix = testing/' /etc/swift/swift.conf"
             # yamllint disable-line rule:line-length
-            - "sed -i -r '1788d' /root/src/code.engineering.redhat.com/swift/test/unit/common/test_utils.py"
+            - "sed -i -r '/self.assertEqual(expected_args, syslog_handler_args)/d' /root/src/code.engineering.redhat.com/swift/test/unit/common/test_utils.py"
```

Comment:
This would be more robust, but I'm not sure if the test will still pass since it has more assertions. 
My suggestion would be to replace the line with the definition of the test and prepend the skip decorator from unittest, this command seemed to do the trick for me:
`sed -i -r 's/(^.*def test_get_logger_sysloghandler_plumbing\(self.*$)/    \@unittest.skip\n\1/' test/unit/common/test_utils.py`

Reply:
Thank you Joan.
This is basically doing the same as the previous method and all tests were then passing.
Am I missing something?

Reply:
Oh right, I misunderstood a bit the proposal. The other assertions in the test were passing. Then it's all good for me, you can ignore the comment, sorry.

Reply:
No problem.
Thank you for the feedback as always.